tests/python/frontend/pytorch/test_forward.py

```diff
@@ -448,6 +448,33 @@ def forward(self, *args):
                  input_data=torch.randn((1, 8, 16, 16)))
 
 
+def test_forward_conv_transpose():
+    torch.set_grad_enabled(False)
+    input_shape = [1, 3, 10, 10]
+
+    class ConvTranspose2D1(Module):
```

Comment:
New tests shouldn't introduce these wrapper modules. You ca use `torch.nn.ConvTranspose2d` directly. No need to have softmax. See

https://github.com/apache/incubator-tvm/blob/3aabbd9c30d247a31eb19ebf997d6074b14b5dd9/tests/python/frontend/pytorch/test_forward.py#L740-L742

Reply:
Ah good, the wrapperless version is much cleaner.